### PR TITLE
Add ESM/CJS double-publishing

### LINKED ISSUE #25: ESM support
Would you like to add dual publishing of CJS/ESM for this package?

### PR DESCRIPTION
**Pull request by kibertoad**
Fixes #25 

OK, that was easier than expected! I would appreciate it if you could publish a prerelease version of this to npm, so that I could test if it works as expected.

### CODE REVIEW

File: package.json
Comment (kibertoad) [inline package.json:25]: index.js was moved to src because that's where unbuild assumes files are, this allows to use a zero-configuration setup. Alternatively I could add explicit configuration

File: package.json
Comment (kibertoad) [inline package.json:27]: I've removed explicit type, because it can actually be easier ESM or CommonJS, but we can't default it to ESM, because otherwise tests break, as older TAP does not support CJS format, and older Node versions don't support ESM

**Comment by kibertoad** [comment]
@brianloveswords How can we proceed with this one?

**Comment by brianloveswords** [comment]
@kibertoad hey sorry for the delay, it's been a good many years since I've published an npm package and I didn't want to mess it up, I started refreshing myself on those instructions before I got distracted.

If you're more seasoned at this, I'm happy to make you a maintainer, but I also don't wanna burden you with that if you're not interested. let me know and I'll either give you the permissions, or I'll get to it later this week!

**Comment by kibertoad** [comment]
@brianloveswords I can do the publishing if you don't mind giving the access; I'm already maintaining fair amount of packages, what is one more :D

**Comment by brianloveswords** [comment]
@kibertoad fantastic, I've added you on npm and in GitHub. also, I did not realize it's getting _this_ many downloads 🤯
image

**Comment by kibertoad** [comment]
oh yeah, your library definitely found its niche!

**Comment by kibertoad** [comment]
@brianloveswords OK, after several iteration this finally works as expected. See https://arethetypeswrong.github.io/?p=buffer-crc32%401.0.0-RC9 report.

**Comment by kibertoad** [comment]
@brianloveswords After this is approved and merged, we are good to release a new version. Do you think that it's time for a big 1.0.0 release?

**Review by brianloveswords — APPROVED** [review]

**Comment by brianloveswords** [comment]
> Do you think that it's time for a big 1.0.0 release?

@kibertoad yeah, I think so! 🎉

**Comment by kibertoad** [comment]
Awesome! should I release it already?

**Comment by brianloveswords** [comment]
ship it! 🚢

**Comment by kibertoad** [comment]
@brianloveswords Aaaaaand it's up!

**Pull request merged**
